src/constants/DefaultFields.ts

```diff
@@ -127,6 +141,12 @@ export const DefaultFamilyTableFields: DefaultField[] = [
   DefaultFields.PREFERRED_CONTACT,
 ];
 
+export const FamilyEnrolmentFields: DefaultField[] = [
+  DefaultFields.CURRENT_PREFERRED_CLASS,
+  DefaultFields.ENROLLED,
+  DefaultFields.CURRENT_CLASS,
+];
```

Comment:
nit: since we're not using this anywhere else, can we just define this in MainRegistration?

Reply:
would also be nice to make the order enrolled, preferred class, current class

Reply:
I was planning to use them in the details sidebar in the future

Reply:
Why the order `enrolled, preferred class, current class` vs alphabetical?

Reply:
the order that they appear in this array determines the order that the table columns show up! that's why I'm thinking we should put it in MainRegistration, so we don't get that confused

Reply:
another thing is we display different enrolment fields on different pages ([Sessions.tsx](https://github.com/uwblueprint/project-read-frontend/blob/bef7ea4c4cfe58e73fc1c8cd21562fd6ff6b6504/src/pages/Sessions.tsx#L178-L182)) so the naming of this also isn't 100% accurate

Reply:
Ok true, I'll move it to MainRegistration and rename it to `FamilyTableEnrolmentFields`. However, I'm  wondering, am I looking at outdated designs? Because I'm seeing the order `preferred class, enrolled, current class` rather than the order that was suggested.
<img width="427" alt="Screen Shot 2021-07-06 at 12 24 55 AM" src="https://user-images.githubusercontent.com/21024100/124542007-98c6ff80-ddf0-11eb-9641-d8918d96669b.png">

Reply:
oh shoot you're right... I might've been looking at an outdated one. can we double check with the designers? it made sense to have them side by side, but I trust them for the final call